src/routes/appRoute.js

```diff
@@ -1,12 +1,39 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
 import { getAppRouteHandler } from './appRouteHandler';
 
 /**
+ * This function provides global error handling when there is a 500 error
+ */
+export const onPreResponse = {
```

Comment:
this was moved from being a global `onPreResponse` handler to being app-route-specific